### DISCUSSION TITLE
Вывод монет XCH с Kukoin не зачислилсь на кошелек Chia wallet

### DISCUSSION THREAD

**Discussion by Aleksandr898** [Chia-Network/chia-blockchain#10005] [Support]
Как можно вернуть монеты? использую легкий кошелек Chia , балан показывает 0, с биржи Kukoin монеты отправились

**Comment by Aleksandr898** [comment]
TxHash 0x8511934f9298dc72e379067a3558ee4f2ffeb69e42b61a07c3d6e5f22af58638

**Comment by chrisgalll** [comment]
the same thing how do you solve that
